docs/products/clickhouse/reference/supported-interfaces-drivers.rst

```diff
@@ -0,0 +1,33 @@
+Interfaces and drivers supported in Aiven for ClickHouse®
+=====================================================================
+
+Find out what technologies and tools you can use to interact with Aiven for ClickHouse®.
+
+.. _clickhouse-interfaces:
+
+Interfaces (protocols)
+----------------------
+
+Aiven for ClickHouse® supports the following fundamental underlying interfaces (protocols):
+
+* ``HTTPS``
+* ``Native TCP``
+* ``MySQL Interface``
+
+.. topic:: Full list of interfaces (protocols) supported in Aiven for ClickHouse
+   
+   Check out `Drivers and Interfaces <https://clickhouse.com/docs/en/interfaces/overview>`_ for all the details.
+
+Drivers (libraries)
+-------------------
+
+There are a number of drivers (libraries) that use one of :ref:`the fundamental underlying interfaces supported in Aiven for ClickHouse <clickhouse-interfaces>` under the hood. It's up to you to pick up a driver (library) of your choice and use it for connecting to your Aiven for ClickHouse service.
```

Comment:
We have a bunch of guides for using different drivers here https://docs.aiven.io/docs/products/clickhouse/howto/list-connect-to-service.  Might be worth linking to this for information about drivers and libraries, especially from an Aiven perspective.  We can then link to the ClickHouse docs and say something like "for the full list of known drivers and libraries for connecting to ClickHouse" or something.

We can also say that any driver that uses one of the supported protocols, and uses TLS, can be used.